core/templates/pages/exploration-editor-page/exploration-save-and-publish-buttons/exploration-save-and-publish-buttons.component.html

```diff
@@ -44,15 +44,15 @@
                alt="Save Draft">
               &#xE161;
             </i>
-            <span class="oppia-save-draft-button-label oppia-save-publish-button-label">Save Draft</span>
+            <span class="oppia-save-draft-button-label oppia-save-publish-button-label protractor-test-save-draft">Save Draft</span>
             <span class="d-none d-md-block oppia-num-changes-text" ng-if="getChangeListLength()">(<[getChangeListLength()]>)</span>
           </span>
           <span class="oppia-publish-label-container" ng-if="!isPrivate()" title="Publish Changes">
             <i class="material-icons md-18 oppia-save-publish-button-icon"
                alt="Publish Changes">
               &#xE2C3;
             </i>
-            <span class="oppia-save-publish-button-label">Publish Changes</span>
+            <span class="oppia-save-publish-button-label protractor-test-publish-changes">Publish Changes</span>
```

Comment:
Similar to above, instead of `protractor-test-publish-changes`, maybe use `protractor-test-publish-changes-message`?

Reply:
Done